docs/canvas/canvas-present-workpad.asciidoc

```diff
@@ -0,0 +1,30 @@
+[role="xpack"]
+[[canvas-present-workpad]]
+== Present your workpad
+
+When you are ready to present your workpad, use and enable the presentation options.
+
+[float]
+[[view-fullscreen-mode]]
+=== View your workpad in fullscreen mode
```

Comment:
Can we stick this at the bottom as the last step? I figure as the user is reading this they'll click into fullscreen mode but then have to click out to the change the settings and then click back into fullscreen mode.

Reply:
Wouldn't they use fullscreen mode when they are presenting their workpad?

Reply:
Yes, they would, but you can't configure the refresh/autoplay intervals inside of fullscreen mode. 

You don't actually need to move this section. Not everyone will need/want to change these settings before entering fullscrene. Let's leave it as is.